kolibri/plugins/coach/assets/src/views/plan/PlanLessonSelectionContentPreview.vue

```diff
@@ -50,8 +50,11 @@
         return false;
       },
     },
+    beforeDestroy() {
+      this.clearSnackbar();
+    },
```

Comment:
I think this should be okay here because `clearSnackbar` is a core action.

However just a heads-up that (I believe) triggering actions and other logic from `beforeDestroy` hooks is an anti-pattern, because by the time the component is destroyed, app vuex state is generally unpredictable. In general I recommend that `beforeDestroy` only gets used for cleanup like removing event listeners and tearing down third-party libraries (like video.js).

Sometimes it's hard to find a reasonable alternative to `beforeDestroy`, though.